pom.xml

```diff
@@ -113,6 +108,11 @@
       <artifactId>commons-text</artifactId>
       <version>1.3</version>
     </dependency>
+    <dependency>
+      <groupId>com.google.graphite</groupId>
+      <artifactId>gcp-client</artifactId>
+      <version>1.0-SNAPSHOT</version>
```

Comment:
This will need to be changed once first version is released. Current version supports building and testing locally.